pages_bots/toolforge_page.py

```
@@ -6,20 +6,69 @@
+
+def log_it(lang, family):
+    # ---
+    table = User_tables.get((lang, family)) or User_tables.get(("*", family))
+    # ---
+    login_bot, logins_cache2 = LoginWrap(lang, family, logins_cache, table)
+    # ---
+    logins_cache.update(logins_cache2)
+    # ---
+    return login_bot
+
+def MainPage(title, lang, family="wikipedia"):
```

Pylint (3.3.7)

[convention] Missing function or method docstring

(C0116)